tools/benchmark/cmd/root.go

```diff
@@ -48,10 +46,3 @@ func init() {
 	RootCmd.PersistentFlags().UintVar(&totalConns, "conns", 1, "Total number of gRPC connections")
 	RootCmd.PersistentFlags().UintVar(&totalClients, "clients", 1, "Total number of gRPC clients")
 }
-
-func Execute() {
```

Comment:
We can hide RootCmd by changing it to rootCmd and only expose Execute. But I feel it does not really make a lot of difference though.